app/assets/javascripts/angular/controllers/series/show.js.erb

```diff
@@ -0,0 +1,8 @@
+app.controller('SeriesShowCtrl', ['$scope', '$http', '$location', '$routeParams', function($scope, $http, $location, $routeParams){
+
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
```

Comment:
Please check: https://github.com/WDI-HK-7/findmetv/pull/7/files#diff-42e805002727ca0c39d869976171df60R3

either you add an extra slash everywhere or in none:
```js
var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
```
vs
```js
var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
```

I'd recommend for you to add it in your Mac Environment first:
```sh
export URL="http://localhost:3000/"
```

and then, directly reference it everywhere:
```js
$http.get(<%= ENV['URL'] %> + 'series/' + $routeParams.id).success(function(response){
```